spec/features/admin_can_add_items_spec.rb

```diff
@@ -0,0 +1,48 @@
+require 'rails_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.